source/_components/modbus.markdown

```diff
@@ -26,13 +26,15 @@ modbus:
   type: tcp
   host: IP_ADDRESS
   port: 2020
+  timeout: 0.5
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.